.github/actions/build-gstreamer-for-windows/action.yml

```
@@ -31,6 +31,14 @@ inputs:
   s3-upload-path:
     description: 'S3 path to upload to'
     required: false
+  gst-plugins-rs-repo:
+    description: 'Repository to use for gst-plugins-rs'
+    required: false
+    default: 'https://github.com/blinemedical/gst-plugins-rs'
+  gst-plugins-rs-ref:
+    description: 'Ref/tag to use for gst-plugins-rs'
+    required: false
+    default: '0.12.9-lldc.1'
```

I'd prefer to leave these blank and pass them in from the calling side.
```suggestion
  gst-plugins-rs-repo:
    description: 'Repository to use for gst-plugins-rs'
    required: false
  gst-plugins-rs-ref:
    description: 'Ref/tag to use for gst-plugins-rs'
    required: false
```